Snowflake.Data.Tests/IntegrationTests/StructuredTypesWithEmbeddedUnstructuredIT.cs

```diff
@@ -312,13 +312,55 @@ public void TestSelectDateTime(string dbValue, string dbType, DateTime? expected
 
         internal static IEnumerable<object[]> DateTimeConversionCases()
         {
-            yield return new object[] { "2024-07-11 14:20:05", SFDataType.TIMESTAMP_NTZ.ToString(), DateTime.Parse("2024-07-11 14:20:05").ToUniversalTime(), DateTime.Parse("2024-07-11 14:20:05").ToUniversalTime() };
-            yield return new object[] { "2024-07-11 14:20:05 +5:00", SFDataType.TIMESTAMP_TZ.ToString(), null, DateTime.Parse("2024-07-11 09:20:05").ToUniversalTime() };
-            yield return new object[] {"2024-07-11 14:20:05 -7:00", SFDataType.TIMESTAMP_LTZ.ToString(), null, DateTime.Parse("2024-07-11 21:20:05").ToUniversalTime() };
-            yield return new object[] { "2024-07-11", SFDataType.DATE.ToString(), DateTime.Parse("2024-07-11").ToUniversalTime(), DateTime.Parse("2024-07-11").ToUniversalTime() };
-            yield return new object[] { "2024-07-11 14:20:05.123456789", SFDataType.TIMESTAMP_NTZ.ToString(), DateTime.Parse("2024-07-11 14:20:05.1234567").ToUniversalTime(), DateTime.Parse("2024-07-11 14:20:05.1234568").ToUniversalTime()};
-            yield return new object[] { "2024-07-11 14:20:05.123456789 +5:00", SFDataType.TIMESTAMP_TZ.ToString(), null, DateTime.Parse("2024-07-11 09:20:05.1234568").ToUniversalTime() };
-            yield return new object[] {"2024-07-11 14:20:05.123456789 -7:00", SFDataType.TIMESTAMP_LTZ.ToString(), null, DateTime.Parse("2024-07-11 21:20:05.1234568").ToUniversalTime() };
+            yield return new object[]
+            {
+                "2024-07-11 14:20:05",
+                SFDataType.TIMESTAMP_NTZ.ToString(),
+                DateTime.Parse("2024-07-11 14:20:05"),
+                DateTime.Parse("2024-07-11 14:20:05") // kind -> Unspecified
+            };
+            yield return new object[]
+            {
+                "2024-07-11 14:20:05 +5:00",
+                SFDataType.TIMESTAMP_TZ.ToString(),
+                null,
+                DateTime.SpecifyKind(DateTime.Parse("2024-07-11 09:20:05"), DateTimeKind.Utc)
+            };
+            yield return new object[]
+            {
+                "2024-07-11 14:20:05 -7:00",
+                SFDataType.TIMESTAMP_LTZ.ToString(),
+                null,
+                DateTime.Parse("2024-07-11 21:20:05").ToLocalTime()
+            };
+            yield return new object[]
+            {
+                "2024-07-11",
+                SFDataType.DATE.ToString(),
+                DateTime.SpecifyKind(DateTime.Parse("2024-07-11"), DateTimeKind.Utc),
```

Comment:
we could change it to DateTimeKind.Unspecified, because that's how it was before arrow implementation

Reply:
done